### PR TITLE
Add recipe for Chatwoot

### PR DESCRIPTION
**Pull request by maximeMD**
This PR adds a recipe for [Chatwoot](https://www.chatwoot.com/).

### CODE REVIEW

File: recipes/chatwoot/webview.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = Ferdi => {
+  const getMessages = function getMessages() {
+    const unreadBadges = document.querySelectorAll('span.unread');
+    const unreadBadgesArray = [...unreadBadges];
+    const unreadBadgesValues = unreadBadgesArray.map(unreadBadgesArray =>
+      Number.parseInt(unreadBadgesArray.textContent),
```
Comment (vraravam) [inline recipes/chatwoot/webview.js:6]: Please use `Ferdi.safeParseInt` instead of `Number.parseInt`

File: recipes/chatwoot/webview.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    );
+    const unreadMessagesCount = unreadBadgesValues.reduce((a, b) => a + b);
```
Comment (vraravam) [inline recipes/chatwoot/webview.js:8]: Rather than using `map` followed by `reduce`, you can use `reduce` directly and collect the sum.

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by maximeMD** [comment]
I just pushed some changes, could you please check this again ? 😺

File: recipes/chatwoot/webview.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
-module.exports = Ferdi => {
+module.exports = (Ferdi) => {
   const getMessages = function getMessages() {
-    const unreadBadges = document.querySelectorAll('span.unread');
+    const unreadBadges = document.querySelectorAll("span.unread");
```
Comment (vraravam) [inline recipes/chatwoot/webview.js:3]: We prefer single quotes. But, i'll approve this PR as is

**Review by vraravam — APPROVED** [review]

**Comment by kytwb** [comment]
@all-contributors please add @maximeMD for code

**Pull request merged**